examples/whisper.wasm/emscripten.cpp

```diff
@@ -106,6 +106,9 @@ EMSCRIPTEN_BINDINGS(whisper) {
                 whisper_reset_timings(g_contexts[index]);
                 whisper_full(g_contexts[index], params, pcmf32.data(), pcmf32.size());
                 whisper_print_timings(g_contexts[index]);
+                if (params.language != nullptr && strcmp(params.language, "en") != 0) {
+                    free((void*)params.language);
+                }
```

Comment:
I think this will not free the memory if the model is multilingual and the provided language from the UI is "en".

Reply:
Ah, let me take another look at this. Thanks!

Reply:
I've updated this now, please take a look when you get a chance. I've deployed it [here](https://danbev.github.io/whisper.cpp/)